include/ck_tile/core/arch/amd_buffer_addressing.hpp

```diff
@@ -2829,6 +2830,60 @@ __device__ auto amd_transpose_load_to_vgpr(const T* __restrict__ in_ptr)
 }
 #endif
 
+// amd_wave_read_first_lane is the SGPR function from AMD GPU device to load 1 or a series of the
+// memory to the SGPR registers.
+__device__ inline uint32_t amd_wave_read_first_lane(uint16_t v)
```

Comment:
These functions can be useful in other places too. For example, I'd like to replace a similar code in #2836.

However they are defined only for clang >= 20. See See https://github.com/ROCm/composable_kernel/pull/2836/files/#r2377436207

Perhaps they can be moved somewhere else? (like `include/ck_tile/core/arch/utility.hpp`)